packages/autocomplete/src/cache.ts

```diff
@@ -1,17 +1,18 @@
 import * as fs from 'fs-extra'
-import * as moment from 'moment'
 
 export async function updateCache(cachePath: string, cache: any) {
   await fs.ensureFile(cachePath)
   await fs.writeJSON(cachePath, cache)
 }
 
 function _isStale(cachePath: string, cacheDuration: number): boolean {
-  return _mtime(cachePath).isBefore(moment().subtract(cacheDuration, 'seconds'))
+  const past = new Date()
+  past.setSeconds(past.getSeconds() - cacheDuration)
+  return past.getTime() > _mtime(cachePath).getTime()
```

Comment:
I feel this is wrong. `_mtime()` returns an integer, right? I would set the typing of `_mtime` to return an integer rather than `any`